Doc/data/refcounts.dat

```diff
@@ -2255,6 +2652,59 @@ PyThreadState_SetAsyncExc:PyObject*:exc:+1:
 PyThreadState_Swap:PyThreadState*:::
 PyThreadState_Swap:PyThreadState*:tstate::
 
+PyThread_acquire_lock:int:::
+PyThread_acquire_lock:PyThread_type_lock:lock::
+PyThread_acquire_lock:int:waitflag::
+
+PyThread_acquire_lock_timed:PyLockStatus:::
+PyThread_acquire_lock_timed:PyThread_type_lock:lock::
+PyThread_acquire_lock_timed:PY_TIMEOUT_T:microseconds::
+PyThread_acquire_lock_timed:int:intr_flag::
+
+PyThread_allocate_lock:PyThread_type_lock:::
+
+PyThread_create_key:int:::deprecated
+
+PyThread_delete_key:void:::deprecated
+PyThread_delete_key:int:key::
+
+PyThread_delete_key_value:void:::deprecated
+PyThread_delete_key_value:int:key::
+
+PyThread_exit_thread:void:::deprecated
+
+PyThread_free_lock:void:::
+PyThread_free_lock:PyThread_type_lock:lock::
+
+PyThread_GetInfo:PyObject *:::
```

Comment:
Needs an effect. It returns a borrowed reference, I think?
```suggestion
PyThread_GetInfo:PyObject *::0:
```

Reply:
No, a strong one (it constructs a new structure object):

```C
    threadinfo = PyStructSequence_New(&ThreadInfoType);
```